blueprint/api.apib

```diff
@@ -481,6 +481,45 @@ Until the Code Corps platform is open to new organizations, only admin users can
 
     + Attributes (Unprocessable Entity Response)
 
+# Group Password
+
+This endpoint is a parent route for reset and forgot password endpoints
+
+
+## Reset [/reset]
+
+### Reset Password [POST]
+
++ Request
+
+    + Headers
+
+            Accept: application/vnd.api+json
+            Authorization: Bearer <token>
+
++ Response 201 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (Reset Password Response)
```

Comment:
I'm sorry, my bad. I forgot the controller action was actually already implemented. That being said, not that I had another look at it, it seems like it might need fixing. The implementation seems strange to me.

- It accepts a token, a password and a password confirmation.
- It verifies the token
- It creates a changeset.
- If the changeset is valid, it renders the user email, but doesn't really alter the user
- If the changeset is not valid, it renders a 422

I'll create an issue to correct this.

I would say, leave the Reset Password documentation out of this PR and just keep the Forgot Password in. I'll post the issue here when I create it.